### ISSUE TITLE
Removing "files before account reset" folder fails

### ISSUE THREAD

**Issue by elexx** [keybase/kbfs#1759]
This issue might be related to #661 .
When deleting my "files before account reset" folder I get an Input/output error
```
➜ ~ $ ls -la /keybase/private/
total 0
dr-x------ 1 elexx root 0 Aug 24 07:07  elexx
dr-x------ 1 elexx root 0 Aug 24 07:07 'elexx (files before account reset 2017-10-29)'

➜ ~ $ ls -la "/keybase/private/elexx (files before account reset 2017-10-29)/"
ls: cannot open directory '/keybase/private/elexx (files before account reset 2017-10-29)/': Input/output error

➜ ~ $ rm -rf "/keybase/private/elexx (files before account reset 2017-10-29)/"
rm: cannot remove '/keybase/private/elexx (files before account reset 2017-10-29)/': Input/output error
```
Versions:
```
➜ ~ $ keybase --version
keybase version 2.6.0-20180817210538+0b2dcf191f
➜ ~ $ uname -a
Linux elexxBook 4.15.0-33-generic #36-Ubuntu SMP Wed Aug 15 16:00:05 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by strib** [comment]
Try `rmdir`.

**Comment by elexx** [comment]
Thanks for the fast reply, unfortunately I get the same error.

```
➜ rmdir /keybase/private/elexx (files before account reset 2017-10-29)/
rmdir: failed to remove '/keybase/private/elexx (files before account reset 2017-10-29)/': Input/output error
```
whereas `ls private/elexx` works fine:
```
➜ ~ $ ls -la /keybase/private/elexx
total 0
```

**Comment by strib** [comment]
Can you ignore it in the files tab of the Keybase GUI?

**Comment by elexx** [comment]
Yes. And now it's also gone from the folder:
```
➜ ~ $ ls -la /keybase/private/
total 0
dr-x------ 1 alexander root 0 Aug 28 15:31 elexx
```
Is this a bug, do you want me to keep the issue open?

**Comment by elexx** [comment]
Just uploaded the logs. The log ID is 06707deaca9cfb720e51c41c

**Comment by elexx** [comment]
In case it help, I was using `rmdir (GNU coreutils) 8.28` when I got the error.

**Comment by strib** [comment]
@elexx I just merged #1763, which should fix the issue with `rmdir` not working for pre-reset folders, in the next Keybase release.
